build.gradle.kts

```diff
@@ -156,12 +165,46 @@ publishing {
     }
 }
 
-fun getSuffix(): String {
-    if (isSnapshot) {
-        return "-SNAPSHOT_${System.getenv("GITHUB_RUN_NUMBER")}"
-    }
+afterEvaluate {
+    plugins.withId("com.vanniktech.maven.publish.base") {
+        configure<MavenPublishBaseExtension> {
+            coordinates(group.toString(), project.the<BasePluginExtension>().archivesName.get(), version.toString())
+
+            if (System.getenv("MAVEN_USERNAME") != null && System.getenv("MAVEN_PASSWORD") != null) {
```

Comment:
MAVEN_USERNAME & MAVEN_PASSWORD are for https://maven.arbjerg.dev
ideally we would publish to both repos, not sure what this check is for tbh

Reply:
me neither, I copied it from lavaplayer